roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=artifacts#/activerecord) on master @ [7ea67fd5d4966508877fc88ba21bdd498ff162e8](https://github.com/cockroachdb/cockroach/commits/7ea67fd5d4966508877fc88ba21bdd498ff162e8):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:876: 
		Tests run on Cockroach v22.1.0-alpha.4-1061-g7ea67fd5d4
		Tests run against activerecord 6.1.5
		6648 Total Tests Run
		6647 tests passed
		1 test failed
		19 tests skipped
		4 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlIntervalTest#test_interval_type (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #79646 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #79491 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -42,6 +42,7 @@ var activeRecordIgnoreList22_1 = blocklist{
 	"FixturesTest#test_create_fixtures":                                    "flaky - FK constraint violated sometimes when loading all fixture data",
 	"IgnoreFixturesTest#test_ignores_books_fixtures":                       "flaky - FK constraint violated sometimes when loading all fixture data",
 	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                     "flaky - FK constraint violated sometimes when loading all fixture data",
+	"PostgresqlIntervalTest#test_interval_type":                            "flaky",
 }
 
 var activeRecordIgnoreList21_2 = blocklist{

